Library/Homebrew/requirements/x11_requirement.rb

```diff
@@ -5,8 +5,7 @@ class X11Requirement < Requirement
   attr_reader :min_version
 
   fatal true
-  cask "xquartz"
-  download "https://xquartz.macosforge.org"
+  default_formula "linuxbrew/xorg/xorg"
```

Comment:
having a default formula in a non-core tap seems like asking for trouble

Reply:
Yeh, I'd rather avoid `linuxbrew` tap usage for now until we're closer to merging the projects.

Reply:
Okay. How about `default_formula "linuxbrew/xorg/xorg"` in `extend/os/linux/requirements/x11_requirement.rb`?

Reply:
How does that change anything? It's still a non-core (i.e. not Linuxbrew/homebrew-core) tap.